sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.use

```diff
@@ -29,7 +29,7 @@ app-admin/sudo -sendmail
 
 # disable hybrid cgroup as we use the unified mode now
 # use lzma which is the default on non-gentoo systems, enable selinux,
-sys-apps/systemd -cgroup-hybrid curl idn lzma selinux
+sys-apps/systemd -cgroup-hybrid curl idn lzma selinux tpm
```

Comment:
maybe we can also add the pkcs11 flag too? https://github.com/flatcar/Flatcar/issues/1394

Reply:
Can we do this in a follow up? I would first like to test this in combination with the clevis TPM support through the new Flatcar-helper service from the next Flatcar nightly.